src/components/explorer/BiosamplesTable.js

```diff
@@ -0,0 +1,165 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+import { withBasePath } from "../../utils/url";
+import ExplorerSearchResultsTableComp from "./ExplorerSearchResultsTableComp";
+
+const biosampleRender = (individual, record) => {
+    const alternateIds = individual.alternate_ids ?? [];
+    const listRender = alternateIds.length ? " (" + alternateIds.join(", ") + ")" : "";
+    return (
+        <>
+            <Link
+                to={{
+                    pathname: withBasePath(`data/explorer/individuals/${record.individual.id}/biosamples`),
+                    state: { backUrl: location.pathname },
+                }}
+            >
+                {individual}
+            </Link>{" "}
+            {listRender}
+        </>
+    );
+};
+
+const customPluralForms = {
+    Serology: "Serologies",
+};
+
+const pluralize = (word, count) => {
+    if (count <= 1) return word;
+
+    if (customPluralForms[word]) {
+        return customPluralForms[word];
+    } else if (word.slice(-1) !== "s") {
+        return word + "s";
+    }
+
+    return word;
+};
+
+const experimentsRender = (studiesType) => {
+    const experimentCount = studiesType.reduce((acc, study) => {
+        acc[study] = (acc[study] || 0) + 1;
+        return acc;
+    }, {});
+    const formattedExperiments = Object.entries(experimentCount).map(
+        ([study, count]) => `${count === studiesType.length ? "" : count + " "}${pluralize(study, count)}`
+    );
+    return (
+        <>
+            {studiesType.every((s) => s !== null) ? (
+                <>
+                    {studiesType.length} Experiment{studiesType.length === 1 ? "" : "s"}:{" "}
+                    {formattedExperiments.join(", ")}
+                </>
+            ) : (
+                <>—</>
+            )}
+        </>
+    );
+};
+
+const experimentsSorter = (a, b) => {
+    const countExperiments = (studiesType) => {
+        return studiesType.filter((s) => s !== null).length;
+    };
+
+    return countExperiments(a.study_types) - countExperiments(b.study_types);
+};
+
+const sampledTissuesRender = (sampledTissues) => {
+    return sampledTissues.map((m) => m.label)[0];
```

Comment:
We pass a list as an argument, map all its items, but the function only returns index 0.